### DISCUSSION TITLE
오늘 좀 늦을것같습니다

### DISCUSSION THREAD

**Discussion by TaeHyoungKwon** [ThinkAboutSoftware/AcademicConference#310] [General]
회사업무로 한 30-40분 정도 늦을것같습니다
빠르게 끝내고 참석하겠습니다

**Comment by jongfeel** [comment]
알겠습니다.

오늘 참여율이 상당히 저조한데 어떻게 될지 모르겠네요

**Reply by TaeHyoungKwon** [reply]
종필님 죄송합니다 오늘 참가가 힘들것같습니다

늦게 말씀드려서 죄송합니다

**Reply by jongfeel** [reply]
그렇군요 오시길 기다렸는데, 많이 아쉽네요.

야근하느라 힘드셨을텐데 잘 쉬시고 다음에 꼭 뵜으면 좋겠습니다.
